lib/build/Dockerfile

```diff
@@ -1,29 +1,27 @@
 FROM alpine:3.14.2 as base
 LABEL maintainer "Symbolica <dev@symbolica.dev>"
-ARG clangParallelJobs
+ARG llvmParallelJobs
 ENV HOME /home/me
 
-RUN clangParallelJobs="${clangParallelJobs:-$(nproc)}" && \
+RUN llvmParallelJobs="${llvmParallelJobs:-$(nproc)}" && \
     apk add --no-cache binutils && \
     apk add --no-cache --virtual build-deps build-base cmake git ninja python3 && \
-    mkdir -p /tmp/clang/build && \
-    cd /tmp/clang && \
+    mkdir -p /tmp/build && \
+    cd /tmp && \
     git clone --depth 1 --branch llvmorg-12.0.1 https://github.com/llvm/llvm-project.git && \
     cd build && \
     cmake -G Ninja \
         -Wno-dev \
         -DCLANG_VENDOR=Alpine \
-        -DLLVM_ENABLE_PROJECTS=clang \
         -DCMAKE_BUILD_TYPE=MinSizeRel \
+        -DLLVM_BUILD_TOOLS=OFF \
+        -DLLVM_ENABLE_PROJECTS=clang \
+        -DLLVM_INCLUDE_EXAMPLES=OFF \
+        -DLLVM_INCLUDE_TESTS=OFF \
+        -DLLVM_PARALLEL_LINK_JOBS=${llvmParallelJobs} \
         -DLLVM_TARGETS_TO_BUILD=Native \
-        -DLLVM_PARALLEL_LINK_JOBS=${clangParallelJobs} \
-        -DLLVM_ENABLE_EH=ON \
-        -DLLVM_ENABLE_FFI=NO \
-        -DLLVM_ENABLE_LIBCXX=NO \
-        -DLLVM_ENABLE_PIC=YES \
-        -DLLVM_ENABLE_RTTI=YES \
```

Comment:
From the docs:

> Build LLVM with run-time type information. Defaults to OFF.

My understanding is that this is a C++ feature and I don't think we need it / use it currently.